### PR TITLE
Add tabs to match latest mockups

### PR DESCRIPTION
**Pull request by amika-sq**
Addresses https://github.com/TBD54566975/web5-wallet/issues/54

Adds tab bar items for each screen in our mockups. Currently, there is no content for Home, Credentials, Connections, and Activity. 

Profiles does have content, but navigation within that is currently broken (navigation is slightly changing with the latest mockups). Will work on that next!

Simulator Screenshot - iPhone 14 Pro - 2023-08-14 at 13 13 22

### CODE REVIEW

File: src/navigation/tab-bar-icon.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from "react";
```
Comment (shamilovtim) [inline src/navigation/tab-bar-icon.tsx:1]: I don't really think you need this code at all? Seems like this is all just so you share  `focusedName && props.focused ? focusedName : normalName` which isn't really worth the abstraction

Comment (amika-sq) [reply src/navigation/tab-bar-icon.tsx:1]: I'm open to alternatives! I couldn't find a way to do this without needing this code.

Comment (shamilovtim) [reply src/navigation/tab-bar-icon.tsx]: Check latest commit I pushed on this branch

File: src/navigation/TabNavigator.tsx [hunk 2]
```diff
@@ -49,24 +50,39 @@ const TabNavigatorOptions: BottomTabNavigationOptions = {
 };
 
 const HomeScreenOptions: BottomTabNavigationOptions = {
-  tabBarIcon: TabBarIcon.ionicons("home-outline", "home"),
+  tabBarIcon: (props) => {
```
Comment (amika-sq) [inline src/navigation/TabNavigator.tsx:53]: Props isn't typed here. Can you please type them?

Comment (shamilovtim) [reply src/navigation/TabNavigator.tsx:53]: It was typed via BottomNavigationOptions I think? I'll double check

Comment (amika-sq) [reply src/navigation/TabNavigator.tsx:53]: Oh, perfect. So long as we have autocomplete, I'm good with it!

**Pull request merged**